### PR TITLE
test: Move test resources for analytics, geo and storage to v2

### PR DESCRIPTION
**Pull request by royjit**
*Issue #, if available:*

*Description of changes:*

*Check points: (check or cross out if not relevant)*

- [ ] Added new tests to cover change, if needed
- [ ] Build succeeds with all target using Swift Package Manager
- [ ] All unit tests pass
- [ ] All integration tests pass
- [ ] Security oriented best practices and standards are followed (e.g. using input sanitization, principle of least privilege, etc)
- [ ] Documentation update for the change if required
- [ ] PR title conforms to conventional commit style
- [ ] If breaking change, documentation/changelog update with migration instructions

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/integ_test.yml [hunk 1]
```diff
@@ -27,6 +27,14 @@ jobs:
           aws_role_to_assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
           aws_region: ${{ secrets.AWS_REGION }}
           aws_s3_bucket: ${{ secrets.AWS_S3_BUCKET_INTEG }}
+      
+      - name: Verify copy resources for v2
```
Comment (ruisebas) [inline .github/workflows/integ_test.yml:31]: Do we still need the previous "Verify copy resources" step on this branch?

Won't the new V2 step just override everything anyway?

Comment (royjit) [reply .github/workflows/integ_test.yml:31]: we still need those for datastore and api, so keeping it for now. Will remove them when all the plugins are moved to v2 sturcture.

Comment (ruisebas) [reply .github/workflows/integ_test.yml:31]: But datastore and api are not run on this workflow 🤔 

Just to clarify, I'm taking about removing the _step_ from this job, not removing the actual resources from AWS

Comment (royjit) [reply .github/workflows/integ_test.yml]: ah got it! thanks for clarifying, removed the old step

**Review by ruisebas — APPROVED** [review]

**Pull request merged**